README.md

```diff
@@ -66,9 +66,13 @@ server.vm.synced_folder 'assets', '/assets', disabled: true
 
 Run 'vagrant up' from your machine (or 'vagrant provision' if your vbox is already running), then:
 
-On an OSX run:  `sudo mount -o resvport 192.168.50.5:/assets /assets`
+1. At the command line, cd into the vampd root directory and mkdir assets
```

Comment:
I see. Better, then, to change new line 69 to:

"At the command line, mkdir /assets."

This would put the 'assets' directory in the user's home directory instead of making it a subdirectory of $vampd, and we could leave the OSX command as is.